drug-interaction/cli/drug-interaction.psl

```diff
@@ -0,0 +1,12 @@
+1: ATCSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: SideEffectSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: GOSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: ligandSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: chemicalSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: seqSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+1: distSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
+
+//prior
```

Comment:
Comments in code should always have a space after the comment token.

Also, some more comment notes that do not apply in this situation, just FYI:
 - Comments should typically be complete sentences.
 - In-line comments that occur after code should have two spaces before the comment token, e.g. `foo();  // Call foo.`.